.travis.yml

```diff
@@ -19,7 +19,8 @@ branches:
 
 language: node_js
 node_js:
-  - 10
+  - 12
+  - 14
```

Comment:
This triggers two separate jobs to be run on Travis CI (one with node 12 and the other with node 14). Do you think we really need this? :thinking: 

![](https://user-images.githubusercontent.com/1061942/111466197-453a8400-8723-11eb-815d-c8da9034472c.png)

Reply:
Hmm, and it seems v12 fails :thinking: Tried two times, both red, restarted 3rd time and we will see.

Reply:
I couldn't reproduce it using the same env as in failed tests (Safari `11.1.2`, Node `12`, React `16.3.2`). However it's unsure if in all 3 runs the errors were the same. I've pushed a commit with logging improvements so we'll see how it goes.
As for the very idea of checking both node versions - in a perfect world I think it would be totally justified. Unfortunately it 1) doubles the time needed for CI to finish checks (for now we can rarely run both builds simultaneously) and 2) seems so raise false alarms, so I'm not sure if we actually want it.